includes/utils.inc

```diff
@@ -1104,6 +1104,14 @@ function wf_crm_get_fields($var = 'fields') {
         'name' => t('Contribution Source'),
         'type' => 'textfield',
       ];
+      // Custom field for contribution contact.
+      $fields['contribution_contribution_contact_id'] = [
```

Comment:
@omarabuhussein Here I updated the machine name for new field as it was conflicting with `contact_id` key.
Have added the contribution as prefix so in params we are getting the field like this `contribution_contact_id`